kubernetes/validators.go

```diff
@@ -245,3 +245,29 @@ func validateAttributeValueIsIn(validValues []string) schema.SchemaValidateFunc
 
 	}
 }
+
+func validateTypeStringNullableIntOrPercent(v interface{}, key string) (ws []string, es []error) {
+	value, ok := v.(string)
+	if !ok {
+		es = append(es, fmt.Errorf("expected type of %s to be string", key))
+		return
+	}
+
+	if value == "" {
+		return
+	}
+
+	if strings.HasSuffix(value, "%") {
+		percent, err := strconv.ParseInt(strings.TrimSuffix(value, "%"), 10, 64)
+		if err != nil {
+			es = append(es, fmt.Errorf("%s: cannot parse '%s' as percent: %s", key, value, err))
+		}
+		if percent < 0 || percent >= 100 {
+			es = append(es, fmt.Errorf("%s: '%s' is not between 0%% and 100%%", key, value))
+		}
+	} else if _, err := strconv.ParseInt(value, 10, 64); err != nil {
```

Comment:
`intstr` does not support 64bit values: https://github.com/kubernetes/apimachinery/blob/kubernetes-1.14.3/pkg/util/intstr/intstr.go#L60

Not sure yet if that validation should be changed.

Reply:
Changed to 32 bit